spec/helpers/ajax-helper.ts

```diff
@@ -139,7 +139,7 @@ export class MockXMLHttpRequest {
   onerror: (e: ErrorEvent) => any;
   onprogress: (e: ProgressEvent) => any;
   ontimeout: (e: ProgressEvent) => any;
-  upload: XMLHttpRequestUpload;
+  upload: XMLHttpRequestUpload = Object.create(null);
```

Comment:
It's not about Type declare... I just want mock the behavior that `listeners after open called would not be fired`. So I just want remove `setter` on `this.upload.progress` here